spec/support/api_spec_helper.rb

```diff
@@ -243,9 +243,7 @@ def expect_result_resource_keys_to_be_like_klass(collection, key, klass)
   end
 
   def expect_result_resources_to_include_keys(collection, keys)
-    expect(response.parsed_body).to have_key(collection)
-    results = response.parsed_body[collection]
-    fetch_value(keys).each { |key| expect(results.all? { |r| r.key?(key) }).to be_truthy, "resource missing: #{key}" }
+    expect(response.parsed_body).to include(collection => all(a_hash_including(*fetch_value(keys))))
```

Comment:
@imtayadeway Fair enough. That is a complex expect line... so if you think it reads better using `a_hash_including` I'm OK.

So LGTM

@gtanzillo I think this is ready to merge, unless you want to wait for @abellotti